systers_portal/meetup/urls.py

```diff
@@ -0,0 +1,26 @@
+from django.conf.urls import url
+
+from meetup.views import MeetupLocationDetailView, MeetupLocationListView
+
+
+urlpatterns = [
+    url(r'^$', MeetupLocationListView.as_view(), name='list_all_meetup_locations'),
```

Comment:
One single PR now includes 7 URL endpoints. It is much harder to review now and it might take several weeks to accept such a PR (bad both for you and for me).

Focus on one single URL endpoint, which should be `/meetup/<slug>/` that lands on the main page of a MeetupLocation. By the way if you plan to use `/meetup/` as a URL to the list of all meetups, I'd recommend renaming it to `/meetups/`, it reflects our intentions better. Then you will have `/meetups/<slug>/` for displaying one single MeetupLocation.